Yandex_LogIn_LogOut_Tests/Constants/Constant.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Yandex_LogIn_LogOut_Tests
+{
+    public class Constant
+    {
+        public static class Url
+        {
+            public static string YANDEX_HOME_PAGE_URL = "https://yandex.by/";
+            public static string YANDEX_PASSPORT_PAGE_URL = "https://passport.yandex.by/auth?origin=home_yandexid&retpath=https%3A%2F%2Fyandex.by&backpath=https%3A%2F%2Fyandex.by";
```

Comment:
This URL is too difficult for assert method. Title validation only will be enough here. Remove this URL.